server/ai_process.go

```diff
@@ -1540,13 +1541,15 @@ func processAIRequest(ctx context.Context, params aiRequestParams, req interface
 		}
 
 		// Don't suspend the session if the error is a transient error.
-		if isRetryableError(err) {
+		sameSessTries++
```

Comment:
I think that at this point we don't know if it's the same session or another session.